Makefile

```diff
@@ -1,198 +1,360 @@
+	echo 'Cleaning up residuals from building of Python package'
+	rm -rf $(BUILD_PYTHON_OUTPUT_DIR)
 
-	python3 -m venv venv
-	venv/bin/pip install -U pip flit
 
 .PHONY: uninstall-python-package
 uninstall-python-package:
-	@echo 'Uninstalling ch-tools'
+	echo 'Uninstalling $(PROJECT_NAME)'
 	rm -rf $(INSTALL_DIR)
 
+
 .PHONY: install-symlinks
 install-symlinks:
-	@echo 'Creating symlinks to /usr/bin/'
+	echo 'Creating symlinks to $(SYMLINK_BIN_DIR)'
 
-	mkdir -p $(DESTDIR)/usr/bin/
+	mkdir -p $(SYMLINK_BIN_DIR)
 	$(foreach bin, chadmin ch-monitoring keeper-monitoring ch-s3-credentials, \
-		ln -sf $(PREFIX)/bin/$(bin) $(DESTDIR)/usr/bin/ ; \
-	)
+		ln -sf $(PREFIX)/bin/$(bin) $(SYMLINK_BIN_DIR);) 
+
 
 .PHONY: uninstall-symlinks
 uninstall-symlinks:
-	@echo 'Removing symlinks from /usr/bin/'
+	echo 'Removing symlinks from $(SYMLINK_BIN_DIR)'
 
 	$(foreach bin, chadmin ch-monitoring keeper-monitoring ch-s3-credentials, \
-		rm -f $(DESTDIR)/usr/bin/$(bin) ; \
-	)
+	    rm -f $(SYMLINK_BIN_DIR)/$(bin);)
+
 
 .PHONY: install-bash-completions
 install-bash-completions:
-	@echo 'Creating bash completions'
+	echo 'Creating bash completions'
 
 	mkdir -p $(DESTDIR)/etc/bash_completion.d/
 	$(foreach bin, chadmin ch-monitoring keeper-monitoring, \
-		cp resources/completion/$(bin)-completion.bash $(DESTDIR)/etc/bash_completion.d/$(bin) ; \
-	)
+	    cp resources/completion/$(bin)-completion.bash $(DESTDIR)/etc/bash_completion.d/$(bin);)
+
 
 .PHONY: uninstall-bash-completions
 uninstall-bash-completions:
-	@echo 'Removing bash completions'
+	echo 'Removing bash completions'
 
 	$(foreach bin, chadmin ch-monitoring keeper-monitoring, \
-		rm -f $(DESTDIR)/etc/bash_completion.d/$(bin) ; \
-	)
+	    rm -f $(DESTDIR)/etc/bash_completion.d/$(bin);)
+
 
 .PHONY: configure-logs
 configure-logs:
-	@echo 'Configuring logging'
+	echo 'Configuring logging'
 
 	mkdir -p $(DESTDIR)/etc/logrotate.d/
 	$(foreach bin, chadmin clickhouse-monitoring keeper-monitoring, \
 		mkdir -p $(DESTDIR)/var/log/$(bin) ; \
 		chmod 775 $(DESTDIR)/var/log/$(bin) ; \
-		cp resources/logrotate/$(bin).logrotate $(DESTDIR)/etc/logrotate.d/$(bin) ; \
-	)
+		cp resources/logrotate/$(bin).logrotate $(DESTDIR)/etc/logrotate.d/$(bin);)
+
 
 .PHONY: uninstall-logrotate
 uninstall-logrotate:
-	@echo 'Removing log rotation rules'
+	echo 'Removing log rotation rules'
 
 	$(foreach bin, chadmin clickhouse-monitoring keeper-monitoring, \
-		rm -f $(DESTDIR)/etc/logrotate.d/$(bin) ; \
-	)
+		rm -f $(DESTDIR)/etc/logrotate.d/$(bin);)
+
 
 .PHONY: prepare-changelog
 prepare-changelog: prepare-version
-	@echo 'Bumping version into Debian package changelog'
-	DEBFULLNAME="Yandex LLC" DEBEMAIL="ch-tools@yandex-team.ru" dch --force-bad-version --distribution stable -v `cat version.txt` Autobuild
+	echo 'Bumping version into Debian package changelog'
+	DEBFULLNAME="Yandex LLC" DEBEMAIL="clickhouse-tools@yandex-team.ru" dch --force-bad-version --distribution stable -v $$(cat $(VERSION_FILE)) Autobuild
```

Comment:
Let's keep `ch-tools@yandex-team.ru` for now. `clickhouse-tools@yandex-team.ru` is nonexistence email.

Reply:
The result of IDE auto-replace.
Thanks for the thorough review.